app/src/test/java/com/mobile/tandil/javabaseproject/DialogFragmentPresenterTest.java

```diff
@@ -0,0 +1,54 @@
+package com.mobile.tandil.javabaseproject;
+
+import com.mobile.tandil.javabaseproject.listener.ListenerDialogFragment;
+import com.mobile.tandil.javabaseproject.mvp.contract.DialogFragmentContract;
+import com.mobile.tandil.javabaseproject.mvp.presenter.DialogFragmentPresenter;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DialogFragmentPresenterTest {
+
+    private DialogFragmentContract.Presenter presenter;
+    private static final String VALID_PARKING = "2401";
+    private static final String EMPTY_STRING = "";
+    private static final String ZERO_PARKING = "0";
+
+    @Mock
+    private DialogFragmentContract.View view;
+
+    @Mock
+    private ListenerDialogFragment dialogFragmentListener;
+
+    @Before
+    public void initialize() {
```

Comment:
the mocks are never initialized. Fix this, please

Reply:
Done